### DISCUSSION TITLE
Expedition 12

### DISCUSSION THREAD

**Discussion by TheRealBeowulf** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#3] [General]
Phase 3 - "Unbounded", doesn't give any rewards (including the one blueprint for Memory of bones) as a result Phase 4 "Deep Glass" (View the third memory), and Phase 5 "Creation" and "Eclipse" can't be completed. This stops the expedition being able to be completed.
Not sure what else to put, thanks for all your effort.

**Comment by cwmonkey** [comment]
And thank you for your report. I'm checking this now - it's possible they changed a reward or changed the name of a reward, etc.

**Reply by cwmonkey** [reply]
Confirmed, Phase 3 - "Unbounded" shows no reward. Gonna look through the rewards tables and see if something matches.

**Reply by cwmonkey** [reply]
oof, I still see the reward listed in the `REWARDTABLE.MBIN` file. Same ID: `RS_S12_S3M6`

In fact the file itself seems to be the same, so it may be a problem with one of the items?

**Reply by cwmonkey** [reply]
Tried swapping the ID to `S12_POI_SEED3` and `R_S12_POI_SEED3` which is the Memory of Bones blueprint, but that also didn't work.

**Reply by cwmonkey** [reply]
Switching the reward to `R_SPACE_WANTED5` which ONLY rewards a Carrier AI Fragment, which I believe is needed for another milestone. That reward shows up, so now I just need to see if I can get that and do the other milestones.

**Reply by cwmonkey** [reply]
Ah, realized you need the recipe to make the item to trigger the next quest. Next thing I'll try is make the item (`POI_SEED_3`) available to buy at space stations.

**Reply by cwmonkey** [reply]
Tried adding `POI_SEED_3` and `S12_POI_SEED_3` to `SeasonalUAStationTradeData.AlwaysPresentProducts` but neither worked. Used a save editor to add the item to my inventory, which didn't unlock the quest. You have to know the recipe to be able to unlock the quest, then you have to have the item to complete the quest.

Going to try to add Memory of Bones to the already known recipes when starting the expedition.

**Reply by cwmonkey** [reply]
Added `S12_POI_SEED3` to `ProductTable` but it didn't make a new save learn it. Last thing I can think to try is replace the "Deep Glass" milestone with some other milestone with the same reward. Hopefully that will put players back on the right path after a mildly confusing detour.

**Reply by cwmonkey** [reply]
That worked, woo. Updated Expedition 12: Omega is here: https://cwmonkey.github.io/nms-expeditions/

**Comment by TheRealBeowulf** [comment]
Thanks for the quick response, is there any way I can manually spawn the item/add the bp to my save in the meantime?.

**Reply by cwmonkey** [reply]
I don't think you actually need the item, I think you just need to claim a reward (any reward). Testing this theory now. Give me a few minutes, not on the fastest computer.

**Reply by cwmonkey** [reply]
Ah dammit, I see now why you need the Memory of Bones blueprint. Well, this is a tough one. Not sure I can fix it without a mod or something 😬

**Reply by cwmonkey** [reply]
The item is `POI_SEED_3` I believe. You'd have to use a save editor and put that in your expedition inventory.

**Reply by cwmonkey** [reply]
Sigh, nope, adding the item to your inventory with a save editor doesn't work. Going to see what happens if I add the item to my known recipes...

**Reply by cwmonkey** [reply]
Yeah that works. You need to know how to make it to unlock the quest, then you need to make it to be able to complete it. Ugh. Using the `NMSSaveEditor` go into "Discovery" and add "Memory of Bones" to "Known Products"

**Comment by cwmonkey** [comment]
OK @TheRealBeowulf, I finally found a solution. If you haven't hacked your save file yet, you can get the replacement JSON for expedition 12 here: https://cwmonkey.github.io/nms-expeditions/

This should work regardless of where you are at in the expedition. Leodium will update his expedition bundle as soon as he's available.
